packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/ContestLiveStep/ContestLiveStep.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import CWPageLayout from 'client/scripts/views/components/component_kit/new_designs/CWPageLayout';
+import { useCommonNavigate } from 'navigation/helpers';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+
+import './ContestLiveStep.scss';
+
+const ContestLiveStep = () => {
+  const navigate = useCommonNavigate();
+
+  return (
+    <CWPageLayout>
+      <div className="ContestLiveStep">
+        contest live
```

Comment:
```diff
- contest live
+ Contest Live
```
?

Reply:
it's just a placeholder, will be changed anyways